### PR TITLE
FSET-1664. Assessor edits their availability

### PR DESCRIPTION
**Pull request by vglushak**

### CODE REVIEW

File: app/services/assessoravailability/AssessorService.scala [hunk 1]
```diff
@@ -44,63 +44,90 @@ trait AssessorService {
...
-    assessorList <- assessorRepository.findAvailabilitiesForLocationAndDate(location, date, skills)
-  } yield assessorList.map { assessor =>
-      model.exchange.Assessor.apply(assessor)
+  def findAvailabilitiesForLocationAndDate(
+    locationName: String, date: LocalDate, skills: Seq[SkillType]
+  ): Future[Seq[model.exchange.Assessor]] = {
+    for {
+      location <- locationsWithVenuesRepo.location(locationName)
+      assessorList <- assessorRepository.findAvailabilitiesForLocationAndDate(location, date, skills)
+    } yield assessorList.map(assessor => model.exchange.Assessor.apply(assessor))
```
Comment (kjharkin) [inline app/services/assessoravailability/AssessorService.scala]: You may as well make this ```model.exchange.Assessor(assessor)```

File: app/services/assessoravailability/AssessorService.scala [hunk 2]
```diff
@@ -44,63 +44,90 @@ trait AssessorService {
...
   def findAssessor(userId: String): Future[model.exchange.Assessor] = {
-    for {
-      assessorOpt <- assessorRepository.find(userId)
-    } yield {
-      assessorOpt.fold(throw AssessorNotFoundException(userId)) {
-        assessor => model.exchange.Assessor.apply(assessor)
-      }
+    assessorRepository.find(userId).map {
+      case None => throw AssessorNotFoundException(userId)
+      case Some(assessor) => model.exchange.Assessor.apply(assessor)
```
Comment (kjharkin) [inline app/services/assessoravailability/AssessorService.scala]: style - why specific ```apply```?

Comment (vglushak) [inline app/services/assessoravailability/AssessorService.scala]: indeed.

**Comment by kjharkin** [comment]
Nice :thumbsup:

**Comment by vglushak** [comment]
Comments addressed.

**Pull request merged**